### PR TITLE
Update about description

### PR DESCRIPTION
**Pull request by canyon289**
Heres the proposal. More action oriented and value to reader, and less about my feelings

### CODE REVIEW

File: dev/content/pages/About.md [hunk 1]
```diff
@@ -105,15 +105,18 @@ save_as: index.html
...
+    <p>I am a data scientist at Google. My focus is supporting strategic decision making with data, although all my opinions here are my own. Previously worked at Sweetgreen and SpaceX, doing the same, using data to inform staffing strategy and rocket launch cadence.</p>
+    
+    <p>To become a better "decision" scientist I'm currently focusing on three topics</p>
     <ol>
-        <li>An expert computational mathematician</li>
-        <li>An effective community member and leader</li>
-        <li>A knowledge contributor</li>
+        <li>Organizational Influence and Leadership</li>
+        <li>Advanced applied mathematics e.g. Causal Inference, Decistion Theory</li>
+        <li>Business Domain expertise</li>
```
Comment (sarinac) [inline dev/content/pages/About.md]: 1. The capitalization seemed kinda inconsistent, so I lower-cased everything if the word wasn't the starting one.
2. Personally, I feel a little iffy stating "business domain expertise" in a non-work/personal setting for two reasons: First, I think of business/domain expertise as company-specific or confidential. Second, usually you need to be an expert in a specific field and you are not stating any field. Could it be more like "business applications/case studies of businesses using statistics to drive a decision", or "domain expertise in <SOMETHING>"? I feel like you will have an easier time going with the former option.
```suggestion
        <li>Organizational influence and leadership</li>
        <li>Advanced applied mathematics, e.g. causal Inference, decision theory</li>
        <li>Business domain expertise</li>
```

File: dev/content/pages/About.md [hunk 2]
```diff
@@ -105,15 +105,18 @@ save_as: index.html
...
     </ol>
-    <p>I always thought it was cool that people would come together and use math to build awesome things, whether it be rocket ships or computational statistics libraries. One of my most rewarding experiences came from contributing to the open source community. Since then, I have been a core contributor to <a href="https://arviz-devs.github.io/arviz/index.html">ArviZ</a> and <a href="https://docs.pymc.io/">PyMC3</a>.</p>
-    <p>I believe in the possibility of creating great outcomes with math. The variety and volume of data has increased faster than the knowledge of how to make full use of it, and I am working to fill the gap. You'll see my public efforts in my open source profile, YouTube channel, book on Bayesian statistics, online course on applied Bayesian statistics, and many more.</p>
+    <p>I like to share and teach the skills I've found useful from my career, whether it be here or in my blog, on YouTube, or as part of my course series</p>
```
Comment (sarinac) [inline dev/content/pages/About.md]: Ravin! I gave you so many periods, semi-colons, and punctuations in some previous review ):<
```suggestion
    <p>I like to share and teach the skills I've found useful from my career, whether it be here or in my blog, on YouTube, or as part of my course series.</p>
```

File: dev/content/pages/About.md [hunk 3]
```diff
@@ -105,15 +105,18 @@ save_as: index.html
...
+    <p>If you'd prefer to updates pushed to you can subscribe to my mailing list </p>
```
Comment (sarinac) [inline dev/content/pages/About.md]: 1. ):<
2. Can you read what you originally wrote? 😂 
3. Here are some options:

Minimal corrections to what you wrote:
```suggestion
    <p>If you'd prefer to have updates pushed to you, you can subscribe to my mailing list.</p>
```
What I prefer:
```suggestion
    <p>If you prefer to get notified whenever I send updates, consider subscribing to my mailing list.</p>
```

File: dev/content/pages/About.md [hunk 4]
```diff
@@ -105,15 +105,18 @@ save_as: index.html
...
+    <p>I also contribute heavily to open source to <a href="https://arviz-devs.github.io/arviz/index.html">ArviZ</a> and <a href="https://docs.pymc.io/">PyMC</a>. </p>
```
Comment (sarinac) [inline dev/content/pages/About.md]:
```suggestion
    <p>I also contribute heavily to open source, mainly with <a href="https://arviz-devs.github.io/arviz/index.html">ArviZ</a> and <a href="https://docs.pymc.io/">PyMC</a>.</p>
```

File: dev/content/pages/About.md [hunk 5]
```diff
@@ -105,15 +105,18 @@ save_as: index.html
...
+    <p>I'm always happy to connect with interested folks. Feel free to reach out anytime</p>
```
Comment (sarinac) [inline dev/content/pages/About.md]: ):< If I get a boba with every punctuation you forget...
```suggestion
    <p>I'm always happy to connect with interested folks. Feel free to reach out anytime.</p>
```

File: dev/content/pages/About.md [hunk 6]
```diff
@@ -105,15 +105,18 @@ save_as: index.html
 <!-- Left -->
 <div class="about-box">
 <h1 style="margin-top:0; text-align: left;">About Me</h1>
-    <p>I am a data scientist at Google where I use data to inform decisions, although all my opinions here are my own. I have previously worked at Sweetgreen, designing systems that would benefit team members and communities through sustainable and healthy food, and SpaceX, creating tools that would ultimately launch rocket ships. I feel fortunate to have worked with amazing teams throughout my career.</p>
-    <p>There are 3 things I strive to be:</p>
+    <p>I am a data scientist at Google. My focus is supporting strategic decision making with data, although all my opinions here are my own. Previously worked at Sweetgreen and SpaceX, doing the same, using data to inform staffing strategy and rocket launch cadence.</p>
+    
+    <p>To become a better "decision" scientist I'm currently focusing on three topics</p>
```
Comment (sarinac) [inline dev/content/pages/About.md]:
```suggestion
    <p>To become a better "decision" scientist, I'm currently focusing on three topics:</p>
```

File: dev/content/pages/About.md [hunk 7]
```diff
@@ -105,15 +105,18 @@ save_as: index.html
 <!-- Left -->
 <div class="about-box">
 <h1 style="margin-top:0; text-align: left;">About Me</h1>
-    <p>I am a data scientist at Google where I use data to inform decisions, although all my opinions here are my own. I have previously worked at Sweetgreen, designing systems that would benefit team members and communities through sustainable and healthy food, and SpaceX, creating tools that would ultimately launch rocket ships. I feel fortunate to have worked with amazing teams throughout my career.</p>
-    <p>There are 3 things I strive to be:</p>
+    <p>I am a data scientist at Google. My focus is supporting strategic decision making with data, although all my opinions here are my own. Previously worked at Sweetgreen and SpaceX, doing the same, using data to inform staffing strategy and rocket launch cadence.</p>
```
Comment (sarinac) [inline dev/content/pages/About.md]: I mainly wanted to corrected "Previously, I" to make it an actual grammatical sentence but threw in other suggestions.
```suggestion
    <p>I am a data scientist at Google. My focus is supporting strategic decision making with data, although all my opinions here are my own. Previously, I worked at Sweetgreen and SpaceX, doing the same line of work as now but on different applications, which is using data to inform staffing strategy and rocket launch cadence.</p>
```

**Review by sarinac — CHANGES_REQUESTED** [review]

File: dev/content/pages/About.md (hunk 1 above)
Comment (canyon289) [reply dev/content/pages/About.md]: Agree I'll change it

File: dev/content/pages/About.md (hunk 5 above)
Comment (canyon289) [reply dev/content/pages/About.md]: Deal, Ive counted 4 boba so far

File: dev/content/pages/About.md (hunk 6 above)
Comment (canyon289) [reply dev/content/pages/About.md]: Five boba

File: dev/content/pages/About.md (hunk 2 above)
Comment (canyon289) [reply dev/content/pages/About.md]: Im really bad with punctuation. :( its one of my worst skills. I hope you dont hate me. Thank you for help

File: dev/content/pages/About.md [hunk 8]
```diff
@@ -105,15 +105,18 @@ save_as: index.html
...
-        <li>A knowledge contributor</li>
+        <li>Organizational influence and leadership</li>
+        <li>Advanced applied mathematics, e.g. Causal Inference, Decision Theory</li>
+        <li>Something else TBD</li>
     </ol>
-    <p>I always thought it was cool that people would come together and use math to build awesome things, whether it be rocket ships or computational statistics libraries. One of my most rewarding experiences came from contributing to the open source community. Since then, I have been a core contributor to <a href="https://arviz-devs.github.io/arviz/index.html">ArviZ</a> and <a href="https://docs.pymc.io/">PyMC3</a>.</p>
-    <p>I believe in the possibility of creating great outcomes with math. The variety and volume of data has increased faster than the knowledge of how to make full use of it, and I am working to fill the gap. You'll see my public efforts in my open source profile, YouTube channel, book on Bayesian statistics, online course on applied Bayesian statistics, and many more.</p>
+    <p>I like to share and teach the skills I've found useful from my career, whether it be here or in my blog, on YouTube, or as part of my course series</p>
```
Comment (sarinac) [inline dev/content/pages/About.md]: 😅 
```suggestion
    <p>I like to share and teach the skills I've found useful from my career, whether it be here or in my blog, on YouTube, or as part of my course series.</p>
```

File: dev/content/pages/About.md [hunk 9]
```diff
@@ -105,15 +105,18 @@ save_as: index.html
...
+    <p>I am a data scientist at Google. My focus is supporting strategic decision making with data, although all my opinions here are my own. Previously, I worked at Sweetgreen and SpaceX, doing the same line of work as now but on different applications, which is using data to inform staffing strategy and rocket launch cadence.</p>
+    
+    <p>To become a better "decision" scientist, I'm currently focusing on three topics:</p>
     <ol>
-        <li>An expert computational mathematician</li>
-        <li>An effective community member and leader</li>
-        <li>A knowledge contributor</li>
+        <li>Organizational influence and leadership</li>
+        <li>Advanced applied mathematics, e.g. Causal Inference, Decision Theory</li>
+        <li>Something else TBD</li>
```
Comment (sarinac) [inline dev/content/pages/About.md:114]: Ok I didn't mean for you to get rid of it completely D: The business-related idea was a good one but not sure the best way to phrase it. 

Some ideas I can think of are (still pretty raw):
* Approaches to problem solving for businesses
* Combining multiple perspectives for problem solving

**Comment by canyon289** [comment]
Apologies for the lack of context. More generally I put this out as a draft to see if this action oriented tone was preferable. IF you think its worse, I wont use it. Just wanted to get your take

File: dev/content/pages/About.md (hunk 9 above)
Comment (canyon289) [reply dev/content/pages/About.md:114]: More spitballing, deep expertise in the "whatever your company does?"

File: dev/content/pages/About.md (hunk 8 above)
Comment (canyon289) [reply dev/content/pages/About.md]: Add another boba to the list. I am genuinely going to pay this out :D. I'll get you a gift card

**Comment by sarinac** [comment]
Going to merge to see how it looks in homepage. I think the action-oriented approach seems better in the long run but might propose some changes later (still TBD on #3 in your list for "deep expertise in the whatever your company does"). Thanks!

**Comment by canyon289** [comment]
Sounds good

**Pull request merged**